CHANGELOG.md

```diff
@@ -1,29 +1,40 @@
 # Changelog
 
+## [0.10.0](https://github.com/haskell-nix/hnix/compare/0.9.1...0.10.0) (2020-09-12)
+
+* Breaking:
+  * [Remove `ToJSON` and `FromJSON` instances for `NExpr`](https://github.com/haskell-nix/hnix/pull/699)
+    * This also removes the JSON output feature for unevaluated expression trees.
+
+* Misc:
```

Comment:
A `Misc` heading IMHO signals that the following sections are unimportant. The `--version` feature IMHO deserves more attention.

Reply:
But the `Other changes` you propose is literally equivalence to the `Miscellaneous` category.

The name `Other changes` signals that the following sections are unimportant just as well also. Because if there is something important -  there is all "Other/Misc".

That is why instead of them I used the category name `Additional` initially, but that got argued in favor of `Other changes`, and now `Misc` is argued also, while being an equivalent.

While initially proposed "Additional" means it sort of important changes, made changes are additions, what changes were `Add`ed by this release. And people love to read what they as a bonus `additionally` got.

Reply:
@Anton-Latukha I never proposed to name a section `Other changes`. It's a fine and conventional name though.

`Additions` is a conventional section in changelogs, and the proper place for new features.